modules/overviews/lib/overviews/engine.rb

```diff
@@ -47,7 +47,8 @@ class Engine < ::Rails::Engine
         OpenProject::AccessControl.permission(:view_project)
           .controller_actions
           .push(
-            "overviews/overviews/show"
+            "overviews/overviews/show",
+            "overviews/overviews/project_life_cycles_sidebar"
```

Comment:
It surprised me that the side bar is part of `view_project`. If I understand it correctly, you must have both `view_project` and `view_project_stages_and_gates` permissions to see the sidebar.

Isn't this superfluous as you wouldn't see the entire overview page to begin with if you lacked the `view_project` permission? 🤔 

In other words: is it necessary to add this line?

Reply:
Yep, it is superfluous, I think it is just a typo. Thanks!